bitrise.yml

```diff
@@ -0,0 +1,63 @@
+---
+format_version: '13'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+
+trigger_map:
+- push_branch: main
+  workflow: build
+- pull_request_source_branch: "*"
+  workflow: build
+
+app:
+  envs:
+  - BITRISE_PROJECT_PATH: Backyard Birds.xcodeproj
+  - BITRISE_SCHEME: Backyard Birds
+  - BITRISE_DISTRIBUTION_METHOD: development
+
+meta:
+  bitrise.io:
+    stack: osx-xcode-15.3.x
+    machine_type_id: g2-m1.8core
+
+workflows:
+  build:
+    summary: Build your Xcode project.
+    description: The workflow will first clone your Git repository, cache and install
```

Comment:
The workflow will first clone your Git repository, install your project's dependencies if any, and then run your project's tests.